lib/tessel/wifi.js

```diff
@@ -38,6 +38,12 @@ Tessel.prototype.findAvailableNetworks = function() {
 
         return networks;
       }, []).sort(compareBySignal);
+    })
+    .catch(function checkNoSuchDeviceError(err) {
+      if (/No such wireless device/.test(err.message)) {
+        return Promise.reject(new Error('Cannot scan; enable WiFi first using "t2 wifi --on"'));
```

Comment:
So "Cannot scan" might be odd feedback for someone who just entered `t2 list` and expected a list of wireless networks. Maybe something a bit more verbose will clear up why that command did not work. 

"Unable to find wireless networks while wifi is disabled. Run "t2 wifi --on" before trying again."

Pinging @Frijol for feedback on this copy as well.